### ISSUE TITLE
Improve accuracy

### ISSUE THREAD

**Issue by rgbayrak** [neurdylab/physio_QA_dl#2]
Experiment with:
- different neural network architectures using keras
- feature engineering
- different hyperparameters for the model
- improve annotations (the accuracy maybe cannot be improved, we will iterate on this)
